core/transaction/payload/Subscribe.go

```diff
@@ -50,7 +54,7 @@ func (s *Subscribe) Deserialize(r io.Reader, version byte) error {
 	if err != nil {
 		return NewDetailErr(err, ErrNoCode, "[Subscribe], Topic Deserialize failed.")
 	}
-	if version == 1 {
+	if version >= 1 {
```

Comment:
I'd  prefer describe the difference of these two version, if it incompatible, we'd consider drop the lower version consider we still in testnet instead of maintain "old" version

Reply:
It would be incompatible and would require ledger reset, so probably it's better to leave version for now and drop lower versions together with balance/25519

Reply:
The version number should just be a workaround now. Starting from balance model, block and transactions are serialized and deserialized through protobuf and should maintain this compatibility automatically.